apps/native/app/src/messages/is.ts

```diff
@@ -448,9 +448,9 @@ export const is = {
     'Þú getur ekki bætt við skírteinum á þetta tæki.',
   'walletPass.barcodeErrorNotConnected':
     'Ekki er hægt að skanna skírteini nema að tækið sé nettengt.',
-  'walletPass.barcodeErrorFailedToFetch': 'Ekki tókst að sækja barkóða',
```

Comment:
I understood it like Sibbi also wanted to update the status message at the top.

Blocking automerge with this comment until he answers again 👍